components/VotePanel/YouVoted.tsx

```diff
@@ -180,19 +199,37 @@ export const YouVoted = ({ quorum }: { quorum: 'electoral' | 'veto' }) => {
       {(isVoting || isInCoolOffTime) && (
         <div className="items-center justify-center flex w-full gap-5">
           <div className="flex flex-col gap-6 items-center">
-            <SecondaryButton
-              className="min-w-[200px]"
-              isLoading={isLoading}
-              tooltipMessage={withdrawTooltipContent}
-              onClick={() => submitRelinquishVote()}
-              disabled={!isWithdrawEnabled || isLoading}
-            >
-              Relinquish Vote
-            </SecondaryButton>
+            {isMulti ?
+              <Button
+                className="min-w-[200px]"
+                isLoading={isLoading}
+                tooltipMessage={withdrawTooltipContent}
+                onClick={() => submitRelinquishVote()}
+                disabled={!isWithdrawEnabled || isLoading}
+              >
+                Change Vote
+              </Button> 
+            :
+              <SecondaryButton
+                className="min-w-[200px]"
+                isLoading={isLoading}
+                tooltipMessage={withdrawTooltipContent}
+                onClick={() => submitRelinquishVote()}
+                disabled={!isWithdrawEnabled || isLoading}
+              >
+                Relinquish Vote
+              </SecondaryButton>
+            }
+
             {isInCoolOffTime && (
               <div className="text-xs">
-                Warning: If you withdraw your vote now you can only deny the
-                proposal its not possible to vote yes during cool off time
+                {isMulti ?
```

Comment:
just make it same the same thing, i realize there's a discrepancy with dan's design but they mean the same thing.